community/contributing/pr_workflow.rst

```diff
@@ -82,15 +82,15 @@ Godot repo, with your GitHub username as namespace:
 
 You can then *clone* your fork, i.e. create a local copy of the online
 repository (in Git speak, the *origin remote*). If you haven't already,
-download Git from `its website <https://git-scm.com>`_ if you're using Windows or 
-Mac, if you're using Linux install it through your package manager. 
+download Git from `its website <https://git-scm.com>`_ if you're using Windows or
+macOS, or install it through your package manager if you're using Linux.
 
-.. note:: if you are on Windows open Git Bash to type commands. Mac and linux users
+.. note:: If you are on Windows, open Git Bash to type commands. macOS and Linux users
           can use their respective terminals.
 
 To clone your fork from GitHub, use the following command:
 
-::
+.. code-block:: shell
```

Comment:
Instead of this, you can keep `::` and just change `.. highlight:: none` to `.. highlight:: shell` at the top of the page.